vendor/assets/javascripts/data-confirm-modal.js

```diff
@@ -137,6 +137,16 @@
       '</div>'
     );
 
+    // Make sure it's always the top zindex
+    var highest = 1050;
```

Comment:
True, I guess I can set it to a variable that grabs the highest z-index
first and then adds to it.

On Mon, Jan 5, 2015 at 1:53 PM, Marcello Barnaba <notifications@github.com>
wrote:

> In vendor/assets/javascripts/data-confirm-modal.js
> <https://github.com/ifad/data-confirm-modal/pull/18#discussion-diff-22480590>
> :
>
> > @@ -137,6 +137,16 @@
> >        '</div>'
> >      );
> >
> > +    // Make sure it's always the top zindex
> > +    var highest = 1050;
>
> ok, but 1050 is as of today. I would avoid depending on an internal
> bootstrap constant :)
> … <#14abb7180ea4232a_>
>  On 05 Jan 2015, at 19:30, Mark Hunt <notifications@github.com> wrote: In
> vendor/assets/javascripts/data-confirm-modal.js: > @@ -137,6 +137,16 @@ >
> '</div>' > ); > > + // Make sure it's always the top zindex > + var highest
> = 1050; 1050 is what bootstrap modals z-index is set at by default, so if
> you want the modal from the confirm to show up above any other manipulated
> modals, it has to have the highest z-index. \ Reply to this email directly
> or view it on GitHub.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ifad/data-confirm-modal/pull/18/files#r22480590>.
>